src/embedded_flash/syscall.rs

```diff
@@ -59,6 +63,37 @@ fn memop(nr: u32, arg: usize) -> StorageResult<usize> {
     }
 }
 
+fn write_slice_op(ptr: usize, value: &[u8]) -> StorageResult<()> {
```

Comment:
Why not simply call those `write_slice` and `erase_page`? The names shouldn't conflict because the other ones are methods.

Reply:
I saw memop and thought it's similar in spirit. `_op` removed.